src/applications/disability-benefits/526EZ/components/create526EmailForm.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { connect } from 'react-redux';
+
+import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
+
+import { apiRequest } from '../../../../platform/utilities/api';
+import { requestStates } from '../../../../platform/utilities/constants';
+import backendServices from '../../../../platform/user/profile/constants/backendServices';
+
+class EmailForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      submissionStatus: requestStates.notCalled,
+      email: ''
+    };
+  }
+
+  updateEmail = (event) => {
+    this.setState({ email: event.target.value });
+  }
+
+  submitEmail = (event) => {
+    event.preventDefault();
+
+    this.setState({ submissionStatus: requestStates.pending });
+
+    /* apiRequest(`/form526_opt_in?email=${this.state.email}`, */
+    apiRequest('/form526_opt_in',
+      {
+        method: 'POST',
+        headers: {
+          'content-type': 'application/json'
+        },
+        body: JSON.stringify({ email: this.state.email })
+      },
+      () => {
+        this.setState({ submissionStatus: requestStates.succeeded });
+      },
+      () => {
+        this.setState({ submissionStatus: requestStates.failed });
+      });
+  }
+
+  render() {
+    const { loading, isLoggedIn, hasEVSSService } = this.props;
+    const { submissionStatus, email } = this.state;
+    if (loading) {
+      return null;
+    }
+
+    if (!isLoggedIn) {
+      return (
+        <AlertBox
+          isVisible
+          headline="To use our new tool, you’ll need to sign in with your premium My HealtheVet or DS Logon account"
+          content="If you don’t have a verified account, you can create an ID.me account to complete the verification process."
```

Comment:
Should we actually link to idme here?

Reply:
Ideally? Yeah. I don't know that it's a problem like this because it's a temporary page, though.